ntbs-service/Models/Entities/TreatmentEvent.cs

```diff
@@ -37,6 +37,10 @@ public class TreatmentEvent : ModelBase, IHasRootEntity
 
         public int NotificationId { get; set; }
         public virtual Notification Notification { get; set; }
+        public string TbServiceCode { get; set; }
```

Comment:
Same with whitespace here, maybe pair up the prop/nav prop  - but new line between each